lib/content-services/src/lib/dialogs/folder.dialog.scss

```diff
@@ -32,3 +32,14 @@
 .adf-dialog-action-button:disabled > span {
     color: var(--theme-text-color);
 }
+@media screen and (max-width: 380px){
+    .adf-full-width{
+        font-size: 6px;
+    }
+    #title{
```

Comment:
don't use ID values just for styling purposes

Reply:
hi @DenysVuika I have changed the adf-full-width class to new class adf-font-size-input

Reply:
i used class instead of id now.